packages/cli/src/commands/deploy/flightcontrol.js

```diff
@@ -0,0 +1,91 @@
+import path from 'path'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { apiServerHandler } from '@redwoodjs/api-server'
+import { getConfig } from '@redwoodjs/internal'
+
+import { getPaths } from '../../lib'
+
+export const command = 'flightcontrol <side>'
+export const alias = 'fc'
+export const description =
+  'Build, Migrate, and Serve commands for Flightcontrol deploy'
+export const builder = (yargs) => {
+  yargs
+    .positional('side', {
+      choices: ['api', 'web'],
+      description: 'select side to build',
+      type: 'string',
+    })
+    .option('prisma', {
+      description: 'Apply database migrations',
+      type: 'boolean',
+      default: true,
+    })
+    .option('serve', {
+      description: 'Run server for api in production',
+      type: 'boolean',
+      default: false,
+    })
+    .option('data-migrate', {
+      description: 'Migrate the data in your database',
+      type: 'boolean',
+      default: true,
+      alias: 'dm',
+    })
+    .epilogue(
+      `For more commands, options, and examples, see ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/docs/cli-commands#deploy'
+      )}`
+    )
+}
+
+export const handler = async ({ side, serve, prisma, dm: dataMigrate }) => {
+  const rwjsPaths = getPaths()
+
+  const execaConfig = {
+    shell: true,
+    stdio: 'inherit',
+    cwd: rwjsPaths.base,
+    extendEnv: true,
+    cleanup: true,
+  }
+
+  async function runApiCommands() {
+    if (serve) {
+      console.log('\nStarting api...')
+      await apiServerHandler({
+        port: getConfig().api?.port || 8911,
```

Comment:
I updated this because it was broken as is. Assuming you'll be port forwarding. Otherwise, I recommend updating in redwood.toml or just hardcoding here.